docs/pages/fragments/install/cli.mdx

```diff
@@ -10,14 +10,13 @@ import TabItem from "@theme/TabItem";
     { label: 'Mac (Silicon/ARM)', value: 'mac-arm', },
     { label: 'Linux (AMD)', value: 'linux', },
     { label: 'Linux (ARM)', value: 'linux-arm', },
-    { label: 'Homebrew Installation', value: 'homebrew', },
     { label: 'Windows Powershell', value: 'windows', },
   ]
 }>
 <TabItem value="brew">
 
 ```bash
-brew install loft-sh/tap/vcluster
+brew install vcluster
```

Comment:
Thanks for the clarification. I wondered why there were 2 tabs for Homebrew, so I picked the more common "brew install" and planned to follow up. I will update.

`brew install vcluster` does install the latest version. Can you tell me why you recommend `brew install loft-sh/tap/vcluster` instead?

Reply:
The `loft-sh/tap/vcluster` formula is part of our tap and is updated faster than the "official" homebrew tap, as it's part of our release pipelines in goreleaser.

Furthermore, our formula is only installing the statically compiled binaries from the GitHub release section instead of building the CLI from the source as the `vcluster` formula does.
This removes the build dependency of Go on a system and cuts down on install time, all while the installed binaries are cosign-signed, so that's a plus for anyone focused on security.